modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -167,14 +168,35 @@ export default {
       );
     },
   },
-  methods: {
-    resetForm() {
-      this.date = '2019-06-15';
-      this.crop = null;
-      this.pickedPlant = null;
-      this.quantity = 1;
-      this.unit = null;
-      this.comment = '';
+  watch: {
+    async selectedType() {
+      console.log('New crop is ' + this.selectedType.name);
+      const selectedPlant = await fetch(
+        'http://farmos/api/fd2_plant_assets?crop=' + this.selectedType.name
+      );
+      const plant = selectedPlant.json();
```

Comment:
`.json()` is an `async` function so should be called using `await` here.
- This probably seems to work for now without `await`, but it creates a race condition that will be likely source of bugs in the future.